src/compo/pace_aod2ioda.py

```diff
@@ -313,7 +351,7 @@ def main():
         type=str, required=True)
     required.add_argument(
         '--retrieval_method',
-        help="name of retrieval method: uaa, remotap",
+        help="name of retrieval method: uaa, remotap, fmapol",
```

Comment:
```suggestion
        help="name of retrieval method: uaa, remotap, fmapol",
        choices=['uaa', 'remotap', 'fmapol'],
```

this will require one of these 3 arguments specifically... can this be autodetected in the files themselves?

Reply:
Thank you, @BenjaminRuston. I will commit your suggestion.

I think it is possible to detect it from files' attributes, but as I can tell, different product has that info in different group.
`FastMAPOL`: under global attributes `:processing_software = "FastMAPOL" ;`
`RemoTAP`: under attributes of `processing_control` group `:software_name = "RemoTAP" ;`
`UAA`: under attributes of `processing_control` group `:software_name = "OCI Unified Aerosol Algorithm (UAA)" ;`
I prefer users to define it because these attributes may change from time to time whenever they update the product.
And users will know what he/she is doing.